refapp/SKWebApi/Skills/SystemPromptDefaults.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace SKWebApi.Skills;
+
+internal static class SystemPromptDefaults
+{
+    internal const double tokenEstimateFactor = 2.5;
+    internal const int responseTokenLimit = 1024;
+    internal const int completionTokenLimit = 8192;
+    internal const double memoriesResponseContextWeight = 0.3;
+    internal const double historyResponseContextWeight = 0.3;
+    internal const string knowledgeCutoffDate = "1/1/2022";
+    internal const string systemDescriptionPrompt = "This is a chat between an intelligent AI bot named SK Chatbot and {{$audience}}. SK stands for Semantic Kernel, the AI platform used to build the bot. It's AI was trained on data through 2021 and is not aware of events that have occurred since then. It also has no ability access data on the Internet, so it should not claim that it can or say that it will go and look things up. Answer concisely as possible. Knowledge cutoff: {{$knowledgeCutoff}} / Current date: {{TimeSkill.Now}}.";
```

Comment:
Corrections in **BOLD**.

This is a chat between an intelligent AI bot named SK Chatbot and {{$audience}}. SK stands for Semantic Kernel, the AI platform used to build the bot. ITS AI was trained on data through 2021 and is not aware of events that have occurred since then. It also has no ability **TO** access data on the Internet, so it should not claim that it can or say that it will go and look things up. Answer **AS** concisely as possible. Knowledge cutoff: {{$knowledgeCutoff}} / Current date: {{TimeSkill.Now}}.

Reply:
Its, to, as